src/js/Form.js

```diff
@@ -1834,19 +1841,27 @@ define( function( require, exports, module ) {
 
         /**
          * Validates all enabled input fields after first resetting everything as valid.
-         * @return {boolean} whether the form contains any errors
+         * @return {Promise} wrapping {boolean} whether the form contains any errors
          */
         FormView.prototype.validateAll = function() {
+            return this.validateContent( $form );
+        };
+
+        /**
+         * Validates all enabled input fields in the supplied container, after first resetting everything as valid.
+         * @return {Promise} wrapping {boolean} whether the container contains any errors
+         */
+        FormView.prototype.validateContent = function( container ) {
```

Comment:
As `container` is now either the `<form>`, or a `.page`, and because the `.page` could also be a `.question`, we have to `addBack()` quite a few times.

This is a little frustrating, and it's tempting to monkeypatch `jQuery` with something like:
```js
$.fn.findIncludingSelf = function(selector) {
    return this.find(selector).addBack(selector);
}
```